lib/aws_sms/version.rb

```diff
@@ -1,3 +1,3 @@
-module AwsSms
-  VERSION = "0.1.0"
+class AwsSms
+  VERSION = "0.4.0"
```

Comment:
Pushed a few version up to rubygems. Required to change this to a new version for successful deploy